### PR TITLE
add cloud theme

### PR DESCRIPTION
**Pull request by KV0001**

### CODE REVIEW

**Comment by thecodrr** [comment]
File must be named theme.json

**Comment by KV0001** [comment]
> File must be named theme.json

I believe everything should be fine now. 
Thanks!

**Comment by thecodrr** [comment]
@KV0001 another error: you are not adding the theme in the `themes/` directory.

**Comment by thecodrr** [comment]
Fixed it for you.

**Review by thecodrr — APPROVED** [review]

File: themes/cloud/v1/theme.json [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  "license": "GPL-3.0-or-later",
+  "homepage": "https://notesnook.com",
+  "description": "A clear environment where focus remains",
+  "colorScheme": "light",
+  "compatibilityVersion": 1,
+  "authors": [
+    {
+      "name": "Kenneth V.",
+      "email": "support@streetwriters.co",
+      "url": "https://streetwriters.co"
```
Comment (thecodrr) [inline themes/cloud/v1/theme.json:15]: You can remove these as they are optional. Or you can add your own if you can.

Comment (thecodrr) [reply themes/cloud/v1/theme.json:15]:
```suggestion
```

Comment (thecodrr) [inline themes/cloud/v1/theme.json:15]:
```suggestion
      "name": "Kenneth V."
```

**Comment by KV0001** [comment]
> Fixed it for you.

Sorry for the inconvenience. Thank you again!

**Comment by KV0001** [comment]
Currently I'm seeing that the code block after changing color on the NN Theme Builder makes the background of the code blocker disappear ending in on white text in the content. 
Choose a default theme
Create a code block
Change the color accents on Primary --- Accent

Don't know if that's currently a bug on the Them Builder. That's the only thing on this theme that is not working properly. If there any solution. Will be happy to fix it.

File: themes/cloud/v1/theme.json (hunk 1 above)
Comment (thecodrr) [reply themes/cloud/v1/theme.json:15]: Hey, I am not really seeing any change here.

**Comment by thecodrr** [comment]
> Currently I'm seeing that the code block after changing color on the NN Theme Builder makes the background of the code blocker disappear ending on white text in the content. Choose a default theme Create a code block Change the color accents on Primary --- Accent
> 
> Don't know if that's currently a bug on the Them Builder. That's the only thing on this theme that is not working properly. If there any solution. Will be happy to fix it.

The code block theme is changed using the `code-block.css` file which is basically a Prism Theme. You can use one of theirs or create your own: https://github.com/PrismJS/prism-themes

File: themes/cloud/v1/theme.json (hunk 1 above)
Comment (KV0001) [reply themes/cloud/v1/theme.json:15]: Ok, thank you for pointing this out. Will change it right away.

**Comment by KV0001** [comment]
Hopefully everything should be good now.

File: themes/cloud/v1/theme.json [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  "license": "GPL-3.0-or-later",
+  "homepage": "https://notesnook.com",
+  "description": "A clear environment where focus remains",
+  "colorScheme": "light",
+  "compatibilityVersion": 1,
+  "authors": [
+    {
+      "name": "Kenneth V.",
+      "email": "",
+      "url": ""
```
Comment (thecodrr) [inline themes/cloud/v1/theme.json:15]:
```suggestion
      "name": "Kenneth V."
```

**Pull request merged**